BubblyEverAfter/urls.py

```diff
@@ -15,8 +15,17 @@
 """
 from django.contrib import admin
 from django.urls import path, include
+from django.conf.urls import url
+from django.conf import settings
+from django.conf.urls.static import static
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("member/", include("MembershipApp.urls")),
+    path("inventory/", include("inventory.urls")),
 ]
+
+if settings.DEBUG:
+
+    urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
This needs a proper discussion to see how you're gonna handle serving static files in production!